Makefile

```diff
@@ -57,19 +57,20 @@ all: plugins.cfg libr/include/r_version.h
 GIT_TAP=$(shell git describe --tags --match "[0-9]*" 2>/dev/null || echo $(VERSION))
 GIT_TIP=$(shell git rev-parse HEAD 2>/dev/null || echo HEAD)
 ifndef SOURCE_DATE_EPOCH
-GIT_NOW=$(shell date +%Y-%m-%d)
+GIT_NOW=$(shell date +%Y-%m-%d__%H:%M:%S"; echo "branch:"; git branch| cut  -d" " -f 2)
```

Comment:
put the git branch logic in a separate variable and concatenate it here. use this:
```
GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
```